QEfficient/cloud/execute.py

```diff
@@ -38,7 +39,29 @@ def main(
     model_hf_path = hf_download(repo_id=model_name, cache_dir=cache_dir, allow_patterns=["*.json"])
     tokenizer = AutoTokenizer.from_pretrained(model_hf_path, use_cache=True, padding_side="left")
 
-    cloud_ai_100_exec_kv(tokenizer=tokenizer, qpc=qpc_path, device_id=devices, prompt=prompt)
+    if inputs_file_path is not None:
+        try:
+            prompt = []
+            with open(inputs_file_path, "r") as file:
+                for line in file:
+                    prompt.append(line.strip())
+        except FileNotFoundError:
+            print("inputs file not found.")
+
+    qpc_dir_name = qpc_path.strip("/").split("/")[-2]
+    compilation_batch_size = int(qpc_dir_name.split("BS")[0].split("_")[-1])
```

Comment:
We need to handle this differently, It is possible that user used compile API and provided a different directory name than our standard way of naming qpc directory.

Reply:
Updated